app/src/js/components/Rules/add.js

```diff
@@ -45,11 +30,48 @@ ModalBody.propTypes = {
   record: PropTypes.object
 };
 
-const AddRule = ({ rules, ...rest }) => {
+const AddRule = ({ rules, location = {}, dispatch, schema, ...rest }) => {
+  const [defaultValue, setDefaultValue] = useState({
+    name: '',
+    workflow: '',
+    provider: '',
+    collection: {
+      name: '',
+      version: ''
+    },
+    meta: {},
+    rule: {
+      type: '',
+      value: ''
+    },
+    state: 'ENABLED'
+  });
+  const { state: locationState } = location;
+  const { name } = locationState || {};
+  const { rule: ruleSchema } = schema || {};
+  const { map: rulesMap } = rules || {};
+  const isCopy = !!name;
+  let title;
+  (isCopy) ? title = 'Copy a rule' : title = 'Add a rule';
```

Comment:
We could make this more concise with `const title = isCopy ? 'Copy a rule' : 'Add a rule'`

We don't have this for the copy collection but perhaps we should add it? 

Also, I'm wondering if it's worth changing the copy to something like \`Copying rule ${name}\` just to further orient the user, but I don't have a strong opinion either way 🤷‍♀

Reply:
I did think about that, add the "copying rule ${name}" But I was thinking once they get to this page, it's more about the new rule and We should encourage them to change the name anyway

Reply:
Yeah agreed. I wonder if we need to have a title change at all then? Like, regardless, they're still "adding" a rule, so it might be good to indicate that. But again, I don't really have a strong opinion on it.